hivelocity/data_source_product.go

```diff
@@ -54,16 +54,17 @@ func dataSourceProduct() *schema.Resource {
 	}
 }
 
+/* TODO FIX - CURRENTLY BROKEN */
```

Comment:
What's broken about it?

Reply:
The previous endpoint this data source used to use was removed so I changed it to use the closed I could find, to at least fix the compilation problem. However, now it's presenting some marshaling errors. I was under the impression the client library can't make sense of the data the inventory endpoint returns (bad swagger perhaps). Anyways, I thought it would be better if we make a separate ticket to fix this issue.

Reply:
As long as the data is the same for any client's product data source we should be fine.